commands/info.js

```diff
@@ -31,6 +30,23 @@ function getProwessReq(defense, level) {
   return defense * 10 * Math.pow(level, 1 / (0.25)) ;
 }
 
+function parseAttackStat({ Damages, SpecialType }) {
+  if (SpecialType !== 'a') return Damages[0];
+
+  return `${Math.min(...Damages)} - ${Math.max(...Damages)}`;
+}
+
+function parseRespawnTime(respawnTime) {
+  // For bosses with respawnTime = 'Special'
+  if (isNaN(respawnTime)) return respawnTime;
+
+  const h = Math.floor(respawnTime / 3600);
+  const m = Math.floor(respawnTime % 3600 / 60);
+  const s = Math.floor(respawnTime % 3600 % 60);
+  const timeString = `${h}h ${m}m ${s}s`;
+  return timeString;
+}
+
 function getMonster(name) {
   let toReturn;
   for (const [, value] of Object.entries(monsterList)) {
```

Comment:
Would we be able to change to this?
```suggestion
  for (value of Object.values(monsterList)) {
```